manifest.yaml

```diff
@@ -2,12 +2,15 @@ dist:
     module: github.com/observatorium/observatorium-otelcol
     name: observatorium-otelcol
     description: Observatorium OpenTelemetry Collector Distribution
-    version: 0.1.0
+    version: 0.1.1
```

Comment:
nit: would 0.2.0 be more appropriate given the change in components (e.g. auth processor)

Reply:
Done!